### PR TITLE
Pull Mutation Assessor from Genome Nexus

### PR DESCRIPTION
**Pull request by inodb**
Pull Mutation Assessor from genome nexus

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,9 @@
     "updateOncoKbAPI": "npm run fetchOncoKbAPI && npm run buildOncoKbAPI",
     "fetchOncoKbAPI": "curl http://oncokb.org/api/v1/v2/api-docs > src/shared/api/generated/OncoKbAPI-docs.json",
     "buildOncoKbAPI": "node scripts/generate-api.js src/shared/api/generated OncoKbAPI",
+    "updateGenomeNexusAPI": "npm run fetchGenomeNexusAPI && npm run buildGenomeNexusAPI",
+    "fetchGenomeNexusAPI": "curl http://localhost:38080/v2/api-docs?group=variant_annotation > src/shared/api/generated/GenomeNexusAPI-docs.json",
```
Comment (inodb) [inline package.json]: @athum could you point this to the heroku instance https://genome-nexus-mutation-assessor.herokuapp.com

File: src/shared/components/mutationTable/column/MutationAssessorColumnFormatter.tsx [hunk 2]
```diff
@@ -125,33 +131,55 @@ export default class MutationAssessorColumnFormatter
...
+                maData = {
+                    impact: mutationAssessor.functionalImpact,
+                    score: mutationAssessor.functionalImpactScore,
+                    pdb: data[0].linkPdb,
+                    msa: data[0].linkMsa,
+                    xVar: data[0].linkXvar
+                };
+            }
+            // when mutation assessor undefined
+            else {
```
Comment (inodb) [inline src/shared/components/mutationTable/column/MutationAssessorColumnFormatter.tsx]: we usually try to follow styling:
```
if () {
    // code block
} else {
    // code block
}

File: src/shared/components/mutationTable/column/MutationAssessorColumnFormatter.tsx [hunk 3]
```diff
@@ -162,7 +190,7 @@ export default class MutationAssessorColumnFormatter
         const msaLink = MutationAssessorColumnFormatter.maLink(maData.msa);
         const pdbLink = MutationAssessorColumnFormatter.maLink(maData.pdb);
 
-        if (maData.score)
+        if (maData.score !== undefined )
```
Comment (inodb) [inline src/shared/components/mutationTable/column/MutationAssessorColumnFormatter.tsx]: `maData.score` !== undefined, doesn't test for `null`. Is that desired?

Comment (athum) [inline src/shared/components/mutationTable/column/MutationAssessorColumnFormatter.tsx]: I wanted to make sure that scores of 0 would still display, but will add in a case to check for null!

**Pull request closed without merging**